ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/staterootcommitter/StateRootCommitterImplBal.java

```diff
@@ -50,27 +55,69 @@ public Hash computeRootAndCommit(
       final WorldStateKeyValueStorage.Updater stateUpdater,
```

Comment:
Maybe `stateUpdater` should have a more specific name to make it clear what it is an updater of?